docs/config.json

```diff
@@ -332,6 +332,7 @@
         {
           "title": "Access Requests",
           "slug": "/access-controls/access-requests/",
+          "forScopes": ["oss", "enterprise", "cloud"],
```

Comment:
The explanation for this change is described [here](https://github.com/gravitational/teleport/pull/22613/files#diff-2826945929dbebc89582ca9a75dbaf3172362a176b94c3d2eae82ab7e59a24e2R17)